tests/common/helpers/generators.py

```diff
@@ -41,14 +41,15 @@ def route_through_default_routes(host, ip_addr):
     return ret
 
 
-def generate_ip_through_default_route(host):
+def generate_ip_through_default_route(host, exclude_ips=[]):
```

Comment:
>[] [](http://example.com/codeflow?start=56&length=2)

Do not use mutable as default parameter, the behavior is not as expected.
ref: [https://codeql.github.com/codeql-query-help/python/py-modification-of-default-value/](https://codeql.github.com/codeql-query-help/python/py-modification-of-default-value/)

Actually LGTM alert is a true issue.